cookbooks/aws-parallelcluster-environment/spec/unit/resources/lustre_setup_spec.rb

```diff
@@ -185,8 +187,10 @@ def self.setup(chef_run)
       end
 
       it 'can not install lustre' do
-        is_expected.to write_log("FSx for Lustre is not supported in kernel version #{kernel_version} of RHEL #{platform_version}, please update the kernel version")
-          .with(level: :warn)
+        expect { chef_run }.to(raise_error do |error|
+          expect(error).to be_a(Exception)
+          expect(error.message).to include("FSx for Lustre is not supported in kernel version #{kernel_version} of RHEL #{platform_version}, please update the kernel version")
```

Comment:
Isn't there a similar test for Rocky? Maybe it's not really necessary though...

Reply:
Good point, we missed this test for rocky

Reply:
added